Gems/Prefab/PrefabDependencyViewer/Code/Tests/PrefabDependencyTreeGenerationTests.cpp

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <PrefabDependencyViewerEditorSystemComponent.h>
+#include <PrefabDependencyTree.h>
+
+#include <DirectedGraph.h>
+#include <PrefabDependencyViewerFixture.h>
+
+#include <AzCore/std/algorithm.h>
+
+namespace PrefabDependencyViewer
+{
+    using DirectedGraph = Utils::DirectedGraph;
+    using TestComponent = PrefabDependencyViewerEditorSystemComponent;
+    using Outcome       = AZ::Outcome<PrefabDependencyTree, const char*>;
+    using NodeList      = AZStd::vector<Utils::Node*>;
+
+    bool str_equal(const char* expected, const char* result)
```

Comment:
Following other code, the function name should be `StrEqual`.

Reply:
Resolved